tests/model_fields/test_jsonfield.py

```diff
@@ -695,8 +714,11 @@ def test_contains_contained_by_with_key_transform(self):
                 )),
             ),
         ]
+        # PostgreSQL requires a layer of nesting.
+        if connection.vendor != 'postgresql':
```

Comment:
This should be a feature flag as CockroachDB (which tries to emulate PostgreSQL in a lot of ways) has the same restriction.

Reply:
Agreed, Do you have any name suggestion?

Reply:
Proposal at https://github.com/django/django/pull/13255.